templates/package.hamlet

```diff
@@ -17,7 +17,8 @@ $newline never
             <h1>
                #{pn} #
                <span .latest-version>
-                 #{displayedVersion} #
+                 <a href="https://hackage.haskell.org/package/#{pn}">
```

Comment:
Tangential.
There's still some data that I go to hackage for that I'd like to see on stackage.org:

* issue tracker link
* what versions of this package are there?
* when was version x.y.z published?
* what are the constraints for version x.y.z? (revision n?)
* Sometimes I just want to look at the whole .cabal file for a given package/version/revision